internal/testutil/testdata/postgres/alltypes/create-tables.sql

```diff
@@ -135,7 +134,7 @@ INSERT INTO alltypes.all_postgres_types (
     DEFAULT,
     32767,  -- smallint_col
     2147483647,  -- integer_col
-    9223372036854775807,  -- bigint_col
+    922337203685477580,  -- bigint_col
```

Comment:
Max postgres bigint is 9223372036854775807 but when written to S3 and then synced to postgres causes the number to be too large. Need to investigate further

Reply:
Hmm, I don't like this.

Reply:
yeah same